projects/schematics/src/migrations/2_0/constructor-deprecations/data/cart-coupon.component.migration.ts

```diff
@@ -99,4 +99,45 @@ export const CART_COUPON_COMPONENT_MIGRATIONS: ConstructorDeprecation[] = [
       },
     ],
   },
+  {
+    class: CART_COUPON_COMPONENT,
```

Comment:
Could you combine the migrations with the ones above?
So it would work if someone migrates from the the 4 params component, as well as the 6 param component that have cart service and also the 6 param component with active cart service.